core/modules/system/css/system.admin.css

```diff
@@ -234,7 +234,9 @@ span.admin-missing {
     white-space: nowrap;
   }
 }
-
+.system-status-report .duplicate-module-overriding {
```

Comment:
An alternative to styling it more prominently via font-weight would be to wrap the actual path in `<strong>`.
So it not only looks more important, but the markup also marks it as being more important (might be relevant for assistive technologies).

Reply:
Makes sense. Done, thanks!